### PR TITLE
fix: Support semantic colors on icon

### PR DESCRIPTION
**Pull request by darryltec**
## Motivations

The chip (#678) colours its icon when it's disabled or has an error. This adds support tho said colors

### Added

- Exported the IconProps
- Disabled and Critical on surface colors to icon 
image

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/design/src/icons/Colors.css [hunk 1]
```diff
@@ -41,3 +41,11 @@
 .orange {
   fill: var(--color-orange);
 }
+
```
Comment (chris-at-jobber) [inline packages/design/src/icons/Colors.css:44]: @darryltec while we're in here, can we add support for _all_ of the semantic colors here?

Comment (darryltec) [reply packages/design/src/icons/Colors.css:44]: for you @chris-at-jobber ? Of course!

Comment (darryltec) [reply packages/design/src/icons/Colors.css:44]: @chris-at-jobber Tis' has been added

**Comment by chris-at-jobber** [comment]
Testing looks good, even outside of Icon:
image

I can override icons with system colors
image

**Review by chris-at-jobber — APPROVED** [review]
Code and QA look good to me

**Pull request merged**